manifests/m/Microsoft/DevHome/0.1801.625.0/Microsoft.DevHome.installer.yaml

```diff
@@ -0,0 +1,43 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.DevHome
+PackageVersion: 0.1801.625.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.19041.0
+InstallerType: msix
+Commands:
+- devhome.devdiagnostics.exe
+- DevHome.HostsFileEditorApp.exe
+- DevHome.RegistryPreviewApp.exe
+- DevHome.EnvironmentVariablesApp.exe
```

Comment:
Not sure if it makes sense to include these? Not sure if these are intended to be user facing?

They were gathered from the `AppxManifest.xml` and work as intended in terminal.

`AppxManifest.xml`: (Truncated)
```XML
<desktop:ExecutionAlias Alias="devhome.devdiagnostics.exe" />
<desktop:ExecutionAlias Alias="DevHome.HostsFileEditorApp.exe" />
<desktop:ExecutionAlias Alias="DevHome.RegistryPreviewApp.exe" />
<desktop:ExecutionAlias Alias="DevHome.EnvironmentVariablesApp.exe" />
 ```